dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -9,7 +9,6 @@
 package com.datadog.android.core.internal
 
 import android.content.Context
-import androidx.annotation.WorkerThread
```

Comment:
The first commit is indeed from https://github.com/DataDog/dd-sdk-android/pull/973, I cherry-picked it explicitly, because it is on the `develop` branch.